JetStreamCompose/jetstream/src/main/java/com/google/jetstream/presentation/screens/videoPlayer/components/VideoPlayerSeeker.kt

```diff
@@ -0,0 +1,101 @@
+package com.google.jetstream.presentation.screens.videoPlayer.components
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Pause
+import androidx.compose.material.icons.filled.PlayArrow
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.derivedStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusRequester
+import androidx.compose.ui.focus.focusRequester
+import com.google.jetstream.data.util.StringConstants
+
+@Composable
+fun VideoPlayerSeeker(
+    focusRequester: FocusRequester,
+    state: VideoPlayerState,
+    isPlaying: Boolean,
+    onPlayPauseToggle: (Boolean) -> Unit,
+    onSeek: (Float) -> Unit,
+    contentProgressInMillis: Long,
+    contentDurationInMillis: Long
+) {
+    val contentProgress by remember(
+        contentProgressInMillis,
+        contentDurationInMillis
+    ) {
+        derivedStateOf {
+            contentProgressInMillis.toFloat() / contentDurationInMillis
+        }
```

Comment:
I reckon this remember + derivedStateOf is redundant. If I understand it correctly, the `contentProgressInMillis` will change with every progress change and therefore there's not much to remember anyway.

I think this can just be `val contentProgress = contentProgressInMillis / contentDurationInMillis`